compiler/pytest.vim

```diff
@@ -2,10 +2,19 @@
 " Compiler: Py.test <http://pytest.org/>
 " Maintainer: Whyme Lyu <5longluna at gmail.com>
 " URL: https://github.com/5long/pytest-vim-compiler
+" Last change by: Arthur Jaron <hifreeo@gmail.com>
+" Last change: 31.05.2014 
 
 if exists("current_compiler")
   finish
 endif
+
+if has("python3") && has("python")
+  CompilerSet makeprg=py.test-3.4\ --tb=short\ -q
+elseif !has("python3") && has("python")
+  CompilerSet makeprg=py.test-2.7\ --tb=short\ -q
+endif
+
```

Comment:
I think this should be dropped.
It will prevent the detection / usage for https://github.com/tpope/vim-dispatch.

Additionally, with Neovim both Python 3 and 2 are typically available, but the `has` check then starts both of them.